modules/widgets/social-media-icons.php

```diff
@@ -186,13 +222,14 @@ class="widefat"
 		foreach ( $this->services as $service => $data ) {
 			list( $service_name, $url ) = $data;
 			?>
+			
+			<fieldset class="social-media-icons-widget">
+				<legend><?php _e( $service_name, 'jetpack' ); ?></legend>
```

Comment:
That won't work I'm afraid.
https://codex.wordpress.org/I18n_for_WordPress_Developers#Placeholders